drivers/interrupt_controller/exti_stm32.c

```diff
@@ -1,21 +1,19 @@
 /*
  * Copyright (c) 2016 Open-RnD Sp. z o.o.
  * Copyright (c) 2017 RnDity Sp. z o.o.
+ * Copyright (c) 2019 Linaro Limited
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 /**
  * @brief Driver for External interrupt/event controller in STM32 MCUs
  *
- * Based on reference manuals:
- *   RM0008 Reference Manual: STM32F101xx, STM32F102xx, STM32F103xx, STM32F105xx
- *   and STM32F107xx advanced ARM-based 32-bit MCUs
- * and
- *   RM0368 Reference manual STM32F401xB/C and STM32F401xD/E
- *   advanced ARM-based 32-bit MCUs
- *
- * Chapter 10.2: External interrupt/event controller (EXTI)
+ * Driver is currently implemented to support following EXTI lines
+ * STM32F0/STM32L0/STM32L4: Lines 0 to 15. Lines > 15 not supported
+ * STM32F0/STM32L0/STM32L4: Lines 0 to 15. Lines > 15 are not mapped on an IRQ
```

Comment:
This is F0/L0. Correct?

Reply:
Actually this should be 
 * STM32F1/STM32F3: Lines 0 to 15. Lines > 15 not supported
 * STM32F0/STM32L0/STM32L4: Lines 0 to 15. Lines > 15 are not mapped on an IRQ.
Fixed, txs!